templates/Segment_Image_Data_GoogleDriveExample.ipynb

```diff
@@ -0,0 +1,450 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### This is a notebook to format your data for segmentation, run the images through the cloud instance of Mesmer, and then extract marker counts and morphological information from all the cells in your images.  This version also exemplifies Google Drive interoperability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": [
+     "import"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# import required packages\n",
+    "import os\n",
+    "import warnings\n",
+    "import skimage.io as io\n",
+    "import matplotlib.pyplot as plt\n",
+    "import xarray as xr\n",
+    "import numpy as np\n",
+    "\n",
+    "from ark.utils import data_utils, deepcell_service_utils, io_utils, load_utils, io_utils, plot_utils, segmentation_utils\n",
+    "from ark.utils.google_drive_utils import init_google_drive_api, GoogleDrivePath, path_join, drive_write_out, DriveOpen\n",
+    "from ark.segmentation import marker_quantification"
+   ]
+  },
+  {
+   "source": [
+    "### To use Google Drive with ark-analysis, you'll first have to initialize it via `init_google_drive_api` and a universal passcode.  To get access to this passcode and Google Drive, email someone  -\\\\\\_('\\_' )_/-"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "init_google_drive_api(\"passcode\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### All data, images, files, etc. must be placed in the 'data' directory, and referenced via '../data/path_to_your_data'.\n",
+    "\n",
+    "The syntax for creating a `GoogleDrivePath` is very similar to that of making a regular path string.  Some general usage tips:\n",
+    " * Use `path_join` instead of `os.path.join` when combining filepaths\n",
+    " * Use `parent / child` syntax when possible to visually differentiate Drive paths from local paths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "file_path"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# set up file paths\n",
```

Comment:
They're all google drive paths for demonstration purposes.  

`deepcell_input_dir` and a few others should be local in practice since they hold 'mid-processing' stages of data, so if the demonstration is just confusing, I can change that to be more practical.